driver/controller.go

```diff
@@ -210,7 +207,7 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 			"resized_from": int(snapshot.SizeGigaBytes),
 			"resized_to":   int(volumeReq.SizeGigaBytes),
 		})
-		if action != nil && action.Status != doneActionStatus {
+		if action != nil && action.Status != godo.ActionCompleted {
```

Comment:
@timoreimann , is this `action != nil` necessary since we didn't get an error from the `.Resize()` call ? 

Also, here we check for `action != nil` when we don't do the same check on line `1052`. 

It's not really related to your PR but if we could make it consistent it would be better IMO.